sdk/communication/Azure.Communication.Sms/README.md

```diff
@@ -23,6 +23,7 @@ To create a new Communication Service, you can use the [Azure Portal][communicat
 ### Using statements
 ```C# Snippet:Azure_Communication_Sms_Tests_UsingStatements
 using System;
+using System.IO;
```

Comment:
Curious why System.IO is required now?

Reply:
Good question, looks some automatic addition. Removed.